javascript/javascript2/week1/shortword.js

```diff
@@ -0,0 +1,11 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"]; // returns 'ø'
+function findShortest(d) {
+  return d.reduce(function (prevWord, currWord) {
```

Comment:
Your program doesn't work with the following array **const danishWords = ["bil", "plante"];**